stable/grafana/templates/clusterrole.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.rbac.create }}
+{{- if and (or .Values.sidecar.dashboards.enabled .Values.sidecar.datasources.enabled) .Values.sidecar.useCurrentNamespace  }}
```

Comment:
this line can probably just be:
```
{{- if .Values.sidecar.useCurrentNamespace  }}
```

The entire file should probably be wrapped in:
```
{{- if and .Values.rbac.create (or .Values.sidecar.dashboards.enabled .Values.sidecar.datasources.enabled) }}
```